sys/nginx/default.conf.template

```diff
@@ -53,13 +53,25 @@ server {
     add_header X-Frame-Options SAMEORIGIN;
     add_header X-XSS-Protection "1; mode=block";
 
-    resolver 127.0.0.11 ipv6=off;
-    resolver_timeout 2s;
-
     location / {
         try_files $uri /index.php$is_args$args;
     }
 
+    location /total {
+        default_type 'text/plain';
+
+        content_by_lua_block {
+            local redis = require "resty.redis"
+            local red = redis:new()
+            local ok, err = red:connect(redis_host, 6379)
```

Comment:
Is this making a redis connection for each request? if so this should be made globally to minimise connections and just perform the get over an existing connection (or reopen it if gone of course)

Reply:
as per [lua-resty-redis's limitations](https://github.com/openresty/lua-resty-redis#limitations), it is not possible to have a global connection because the functionality is not available in the contexts of `init_by_lua*` :(

Reply:
lua sucks ;P